webrtc-stats.html

```diff
@@ -2646,6 +2649,43 @@ <h2>
                   Represents the total number of <a>RTCP SR</a> blocks sent for this <a>SSRC</a>.
                 </p>
               </dd>
+              <dt>
+                <dfn>roundTripTime</dfn> of type <span class=
+                "idlMemberType">double</span>
+              </dt>
+              <dd>
+                <p>
+                  Estimated round trip time for this <a>SSRC</a> based on an <a>RTCP Sender Report</a> (SR)
+                  that contains an <a>DLRR</a> report block as defined in [[!RFC3611]]. The Calculation of
```

Comment:
"based on an SR" is a bit vague. Should this say "based on the latest SR that contains" instead?

Reply:
I agree. well, if we look a few lines above, see remoteTimestamp section, it uses "an" SR too.

Reply:
"an SR" is okay or use "a Sender Report (SR)" .

Reply:
emm... I essentially agree with Henrik in specifying which SR to use, while I just found the SR is not specified in, for example, remoteTimestamp